### PR TITLE
Added customization for some Telescope menu keys

### PR DESCRIPTION
**Pull request by binarybrain11**
User can now customize the keys set by LeetBuddy to navigate the Telescope menu. Alt keys are sometimes frustrating to use since many terminals and GUIs use Alt keys and don't send them to applications. This pull request allows users to easily specify alternate keybinds in the setup of LeetBuddy. Defaults remain the same, though I was unable to test any of them except for <CR> and <A-y> due to my setup.

### CODE REVIEW

**Comment by binarybrain11** [comment]
I've also added all of the example test cases instead of just the one sample test case.

File: README.md [hunk 1]
```diff
@@ -64,6 +64,19 @@ LeetBuddy.nvim allows you to customize certain aspects of its behavior. You can
...
+        -- These replace Alt with Ctrl
+        reset = "<C-r>",
+        easy = "<C-e>",
+        -- Can't use <C-m> since that is interpreted as <CR>
+        medium = "<C-d>",
+        hard = "<C-h>",
+        accepted = "<C-a>",
+        not_started = "<C-y>",
+        tried = "<C-t>",
+    }
```
Comment (Dhanus3133) [inline README.md:79]:
```suggestion
    keys = {
        select = "<CR>",
        reset = "<C-r>",
        easy = "<C-e>",
        medium = "<C-d>",
        hard = "<C-h>",
        accepted = "<C-a>",
        not_started = "<C-y>",
        tried = "<C-t>",
    }
```

File: lua/leetbuddy/default_config.lua [hunk 2]
```diff
@@ -5,6 +5,16 @@ local default_config = {
...
+  keys = {
+    select = "<CR>",
+    reset = "<A-r>",
+    easy = "<A-e>",
+    medium = "<A-m>",
+    hard = "<A-h>",
+    accepted = "<A-a>",
+    not_started = "<A-y>",
+    tried = "<A-t>",
+  },
```
Comment (Dhanus3133) [inline lua/leetbuddy/default_config.lua:17]:
```suggestion
  keys = {
    select = "<CR>",
    reset = "<C-r>",
    easy = "<C-e>",
    medium = "<C-m>",
    hard = "<C-h>",
    accepted = "<C-a>",
    not_started = "<C-y>",
    tried = "<C-t>",
  },
```

**Review by Dhanus3133 — CHANGES_REQUESTED** [review]
Let the ctrl be the default key to filter the problems.

**Review by Dhanus3133 — APPROVED** [review]

**Comment by Dhanus3133** [comment]
Merged! Thanks for the PR. :+1:

**Pull request merged**